doc/src/Howto_bpm.rst

```diff
@@ -3,16 +3,16 @@ Bonded particle models
 
 The BPM package implements bonded particle models which can be used to
 simulate mesoscale solids.  Solids are constructed as a collection of
-particles which each represent a coarse-grained region of space much
-larger than the atomistic scale. Particles within a solid region are
+particles, which each represent a coarse-grained region of space much
+larger than the atomistic scale.  Particles within a solid region are
```

Comment:
Oh-oh, are we going to have to put two spaces after *every* period?

Reply:
Please no...

Reply:
No. Those are usually inserted automatically by emacs when I reindent or rewrap a block of text. Same as for the class name, this is a "can have" but not a "must have" change.